STEER/STEER/AliGenerator.h

```diff
@@ -179,6 +187,12 @@ class AliGenerator : public TNamed, public AliRndm
     TString      fProjectile;    // Projectile
     TString      fTarget;        // Target
 
+    ///-------------------------------------------------------------------------
+    // Custom trigger
+    Bool_t fSetUserTrig;
+    Bool_t (*fUserTrigger)(AliStack*);
```

Comment:
@lucamicheletti93 please init fSetUserTrig and fUserTrigger to false and 0 resp. in the constructors

Reply:
Included In the last commit!